modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -37,18 +37,20 @@ Databases created on, or migrated to, Neo4j 2025.06 or later will continue to ha
 This is true for link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/#manage-database-systems[system, standard, and composite] Neo4j databases.
 However, it is possible to set a different default language on both new and existing system, standard, and composite databases.
 
-For more information about how to change and view the default language of new and existing databases and remote database aliases, and for how to set the necessary privilege to change the default language of a standard database, see the following links in the Operations Manual:
+For more information about how to change and view the default language of new and existing databases and remote database aliases, and for how to set the necessary privilege to change the default language of a standard or composite database, see the following links in the Operations Manual:
```

Comment:
feels slightly odd to only add privileges for the database cases and not the alias one 🤔 the alias privilege page doesn't talk about default language specifically but `=== Grant privilege to modify aliases` will be the relevant privilege for altering default language on a remote alias 🤷